lugbot.py

```diff
@@ -3,13 +3,20 @@
 from telegram.ext import Updater, CommandHandler, MessageHandler, Filters, InlineQueryHandler
 import configparser
 import logging
-from telegram import ChatAction
+import requests
+import re
+import ast
+from datetime import datetime, timedelta
+from pytz import timezone, utc
+from time import sleep
+from telegram import ChatAction, InlineKeyboardButton, InlineKeyboardMarkup
 
 logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',level=logging.DEBUG)
 
 config = configparser.ConfigParser()
 config.read('bot.ini')
 
+meetupApi={'sign':'true','key':'Meetup-Api-Key'}
```

Comment:
Add 'scroll':'next_upcoming' and 'page':'1' to get only the next meetup instead of the entire upcoming meetup.